src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Righteous&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo__header">
+      <div>
+        <img 
+          src="./images/logo.png" 
+          alt="Moyo"
+          class="moyo__logo"
+        >
+        <a href="#home" class="moyo__logo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#laptops&computers" 
+              class="nav__link laptops__computers" 
+              data-qa="hover"
+            >Laptops & Computers
```

Comment:
Now i have a some problems

Reply:
Siman@Kotiks MINGW64 ~/projects/layout_moyo-header (develop)
$ git add ./src
fatal: Unable to create 'C:/Users/Siman/projects/layout_moyo-header/.git/index.lock': File exists.

Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue.

Siman@Kotiks MINGW64 ~/projects/layout_moyo-header (develop)
$ git push origin develop
To https://github.com/Yaroslav65/layout_moyo-header.git
 ! [rejected]        develop -> develop (non-fast-forward)
error: failed to push some refs to 'https://github.com/Yaroslav65/layout_moyo-header.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Siman@Kotiks MINGW64 ~/projects/layout_moyo-header (develop)

Reply:
I accidentally clicked commit under this comment and now I have a problem